src/Riok.Mapperly/Emit/SyntaxFactoryHelper.cs

```diff
@@ -141,17 +150,30 @@ public static InvocationExpressionSyntax StaticInvocation(IMethodSymbol method,
         return FieldDeclaration(variableDeclaration).WithModifiers(modifierTokenList);
     }
 
-    public static LocalDeclarationStatementSyntax DeclareVariable(string variableName, ExpressionSyntax initializationValue)
+    public static BinaryExpressionSyntax LessThan(ExpressionSyntax left, ExpressionSyntax right)
```

Comment:
Sorry, didn't catch this the last time. Seems unused.

Reply:
Oops removed.